src/components/Attachments/AttachmentCarousel/Pager/AttachmentCarouselPagerContext.ts

```diff
@@ -17,16 +18,28 @@ type AttachmentCarouselPagerItems = {
 };
 
 type AttachmentCarouselPagerContextValue = {
-    /** The list of items that are shown in the pager */
+    /** List of items displayed in the attachment */
```

Comment:
```suggestion
    /** List of attachments displayed in the pager */
```

Is this more accurate?